test/e2e/baseFixture.ts

```diff
@@ -36,6 +41,8 @@ export const describe = (name: string, includeCredentials: boolean, fn: (codeSer
       authenticated: includeCredentials,
       // This provides a cookie that authenticates with code-server.
       storageState: includeCredentials ? storageState : {},
+      // TODO@jsjoeio add note why we do this for testing cert stuff
+      ignoreHTTPSErrors: true,
```

Comment:
@code-asher do you think we should keep this in still?

Reply:
I think if it works without it then we should remove it.

Reply:
>    page.goto: net::ERR_CERT_AUTHORITY_INVALID at https://localhost:40005/

Argh looks like we need it

Reply:
Ah bugger makes sense I suppose.